packages/astro/src/integration/index.ts

```diff
@@ -40,21 +41,24 @@ export const sentryAstro = (options: SentryOptions = {}): AstroIntegration => {
                 sourcemap: true,
               },
               plugins: [
-                sentryVitePlugin({
-                  org: uploadOptions.org ?? env.SENTRY_ORG,
-                  project: uploadOptions.project ?? env.SENTRY_PROJECT,
-                  authToken: uploadOptions.authToken ?? env.SENTRY_AUTH_TOKEN,
-                  telemetry: uploadOptions.telemetry ?? true,
-                  sourcemaps: {
-                    assets: uploadOptions.assets ?? [getSourcemapsAssetsGlob(config)],
-                  },
-                  _metaOptions: {
-                    telemetry: {
-                      metaFramework: 'astro',
+                sentryVitePlugin(
+                  dropUndefinedKeys({
+                    org: uploadOptions.org ?? env.SENTRY_ORG,
+                    project: uploadOptions.project ?? env.SENTRY_PROJECT,
+                    authToken: uploadOptions.authToken ?? env.SENTRY_AUTH_TOKEN,
+                    telemetry: uploadOptions.telemetry ?? true,
+                    sourcemaps: {
+                      assets: uploadOptions.assets ?? [getSourcemapsAssetsGlob(config)],
                     },
-                  },
-                  debug: options.debug ?? false,
-                }),
+                    bundleSizeOptimizations: options.bundleSizeOptimizations,
```

Comment:
h: sorry almost missed this but the vite plugin doesn't yet support `excludeTracing`, right?

I think we need to remap this key specifically to `excludePerformanceMonitoring`